jaxopt/_src/bfgs.py

```diff
@@ -286,7 +292,7 @@ def __post_init__(self):
         value_and_grad=True,
         has_aux=True,
         maxlsiter=self.maxls,
-        max_stepsize=self.max_stepsize,
+        max_stepsize=None,
```

Comment:
Why did you change this line?

Reply:
I thought the max_stepsize was not needed a priori but I can differentiate max_init_stepsize and max_stepsize as said in the other document. If that sounds good to you I'll put back max_stepsize=self.max_stepsize here then.

Reply:
It is confusing for me if we have `max_stepsize` option but we do not propagate it...

Reply:
Kept as it was.